src/assets/Generator.Shared/TypeFormatters.cs

```diff
@@ -19,6 +20,7 @@ internal class TypeFormatters
         {
             "D" => value.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture),
             "U" => value.ToUnixTimeSeconds().ToString(CultureInfo.InvariantCulture),
+            "O" when value.Offset == TimeSpan.Zero => value.ToString("yyyy-MM-ddTHH:mm:ss.fffffffZ", CultureInfo.InvariantCulture),
```

Comment:
Do we need to account for casing here? Both "O" and "o" are valid.

Reply:
yep, done.